Python/bltinmodule.c

```diff
@@ -1303,6 +1303,7 @@ typedef struct {
     PyObject_HEAD
     PyObject *iters;
     PyObject *func;
+    int strict;
```

Comment:
Consider including `stdbool.h` and making this a `bool`.
`git grep stdbool Python/` for inspiration.

Reply:
I see no need to diverge from the equivalent code in `zip` which currently uses `int`:
```
typedef struct {
    PyObject_HEAD
    Py_ssize_t tuplesize;
    PyObject *ittuple;     /* tuple of iterators */
    PyObject *result;
    int strict;
} zipobject;
```

Reply:
> I see no need to diverge from the equivalent code in zip which currently uses int

We've steadily introduced `stdbool.h` and the `bool` type to the CPython source code; `git grep stdbool` for enlightenment. Using `bool` here results in clearer, more readable, and more maintainable code.